index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="/assets/images/logo-tso.png" />

```

Comment:
I think it would look nicer if it was a logo with transparent background.

Reply:
I will change that in the following days, because if we opt for something with a transparent background it will look different on dark mode and light mode (if the icon we have is white, on browser light mode the icon will be invisible and vice versa for dark mode). We can add this as an issue for further development and change it then and go with this version for now if that's ok

Reply:
Okay, I will create a ticket